### DISCUSSION TITLE
Detection training when no box is available

### DISCUSSION THREAD

**Discussion by vankhoa21991** [Project-MONAI/tutorials#1203] [Ideas]
Hi, thanks for a very comprehensive tutorial on detection. However as far as I notice for volume without any box the model can not be trained, that's why you trained your model on 601/888 series of LUNA16. Do you have any plan to upgrade this feature in the future because the ability of training on negative samples are also important.

Thanks

**Comment by Nic-Ma** [comment]
Hi @Can-Zhao ,

Could you please help share some comments about this question?

Thanks in advance.

**Comment by Can-Zhao** [comment]
Thank you @vankhoa21991. I tested it and found the issue is with the data loader. It may take me some time to fix it. But i will work on it.

**Comment by Can-Zhao** [comment]
The issue is resolved with PR https://github.com/Project-MONAI/tutorials/pull/1256.
